cluster-controller/src/main/java/io/strimzi/controller/cluster/operations/cluster/KafkaClusterOperations.java

```diff
@@ -613,8 +613,13 @@ public void update(String namespace, String name, Handler<AsyncResult<Void>> han
     }
 
     @Override
-    public void reconcile(String namespace, Map<String, String> labels) {
-        reconcile(namespace, labels, CLUSTER_TYPE_KAFKA);
+    public void reconcileAll(String namespace, Map<String, String> labels) {
+        reconcileAll(namespace, labels, CLUSTER_TYPE_KAFKA);
```

Comment:
It's a minor point, but it makes more sense to me having the cluster type as the first argument, rather than the last. I think that woudl match `execute()`

Reply:
Agree, I'll do the same for the `reconcile` method as well.